src/astra_driver.cpp

```diff
@@ -158,7 +158,7 @@ void AstraDriver::advertiseROSTopics()
 {
   rmw_qos_profile_t custom_camera_qos_profile = rmw_qos_profile_default;
 
-  custom_camera_qos_profile.depth = 1;
+  custom_camera_qos_profile.depth = 50;
```

Comment:
I haven't quantified it.  I was mostly pushing 50 for everything because that seemed to work around the bug where rclcpp::spin_some() would spin forever.  I can retry with the default sensor profile and see how well that works.